src/structure_managers/structure_manager.hh

```diff
@@ -809,6 +809,10 @@ namespace rascal {
   std::array<int, Order>
   StructureManager<ManagerImplementation>::ClusterRef<Order>::get_atom_types()
       const {
+    /*
+     * TODO(markus)
```

Comment:
I am not sure what this todo is referring to?
Could you please elaborate?

Reply:
I probably meant to document a bit this part on the functionality